src/plugins/muc-views/modals/templates/occupant.js

```diff
@@ -2,10 +2,168 @@ import 'shared/avatar/avatar.js';
 import { __ } from 'i18n';
 import { html } from "lit";
 import { until } from 'lit/directives/until.js';
-import { _converse, api } from "@converse/headless/core";
+import { setRole, verifyAndSetAffiliation } from "../../utils.js"
+import { showOccupantModal } from '../../utils.js';
+import { _converse, api, converse } from "@converse/headless/core";
 
+export const tpl_footer = (el) => {
+    const model = el.model ?? el.message;
+    const jid = model.get('jid');
+    const muc = model?.collection?.chatroom;
+
+    if (!jid || !muc) {
+        return;
+    }
+
+    const role = model.get('role') ?? 'none';
+    const affiliation = model.get('affiliation');
+
+    const ownRole = muc.getOwnRole();
```

Comment:
It turns out that this variable is not needed since only applicable commands are considered,
and mute, kick, etc are not applicable to non moderators. So this variable factors out.